src/init.cpp

```diff
@@ -1361,6 +1362,9 @@ bool AppInitMain(NodeContext& node)
     // which are all started after this, may use it from the node context.
     assert(!node.mempool);
     node.mempool = &::mempool;
+    assert(!node.chainman);
+    node.chainman = &::g_chainman;
```

Comment:
In commit "node: Add chainman alias for g_chainman" (faad46a5358a12691c80c027caf666e91ca85a1e)

Here and in test setup could consider replacing double redundant `::g_` prefix with just `g_`. If a variable name already begins with `g_` then `::` doesn't tell you much and gets in the way of creating namespaces

Reply:
Good point. Will address on the next force push or the follow-up pull, whichever comes earlier.

Reply:
Thanks, fixed.